### PR TITLE
Finishing up offline game

### PR DESCRIPTION
**Pull request by Tsathogguaa**
Finishing up offline game from last week. Testing on the emulator works fine, need to write actual tests, and go over some of the past tests that may need some changes as well.

### CODE REVIEW

File: app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt [hunk 1]
```diff
@@ -91,11 +91,12 @@ class DeleteSongsActivity : AppCompatActivity() {
 
         var currentLine = recordReader.readLine()
         var button: Button = btn as Button
+        val buttonText = button.text.toString()
 
-        while(currentLine != null) {
+        while (currentLine != null) {
```
Comment (laurislopata) [inline app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt:96]: Good readability improvement

File: app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt [hunk 2]
```diff
@@ -125,6 +126,38 @@ class DeleteSongsActivity : AppCompatActivity() {
...
+        var propWriter = BufferedWriter(FileWriter(tempProp, true))
+
+        var currentLine = propReader.readLine()
+        val buttonSplit = buttonText.trim().split("-")
+
+        while (currentLine != null) {
+            var trimmed = currentLine.trim()
+            val split = trimmed.split(" - ")
+
+            if (split[0].trim().lowercase() == buttonSplit[0].trim().lowercase()
```
Comment (laurislopata) [inline app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt:143]: Maybe add some comments explaining what split[0] and split[1] are?

**Review by laurislopata — APPROVED** [review]
Added one comment you could address but overall really goo job :)

File: app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt (hunk 1 above)
Comment (Tsathogguaa) [reply app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt:96]: Got it

File: app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt (hunk 2 above)
Comment (Tsathogguaa) [reply app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt:149]: Done, can you check if the explanations are clear?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -144,6 +144,8 @@
     <string name="online_game_buzzer_title">Online Buzzer</string>
     <string name="join_room">Join party room</string>
     <string name="create_room">Create party room</string>
+    <string name="game_setup_internet_switch_on">Internet is on</string>
+    <string name="game_setup_internet_switch_off">Internet is off</string>
```
Comment (MaximeZmt) [inline app/src/main/res/values/strings.xml:148]: As written in the wiki, consider using string name that respect string convention and put them at the top of the document in the corresponding part. (see here https://github.com/MaximeZmt/SDP_2022-Vibester/wiki)
Meaning

gameSetup_internetSwitchOn
gameSetup_internetSwitchOff

**Review by MaximeZmt — COMMENTED** [review]
Nice work 👍 ! Just left tiny details in comments

File: app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt [hunk 4]
```diff
@@ -125,6 +126,44 @@ class DeleteSongsActivity : AppCompatActivity() {
         return false
     }
 
+    private fun removeFromProperties(buttonText: String): Boolean {
```
[the remark this answers is not in the document]
Comment (Tsathogguaa) [reply app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt:129]: Good catch! Changed it, committing soon!

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (Tsathogguaa) [reply app/src/main/res/values/strings.xml:148]: Understood, will be in the next commit soon

**Review by MaximeZmt — APPROVED** [review]
LGTM 👍 😃

**Pull request merged**